### DISCUSSION TITLE
tasks - show only icons not grouped

### DISCUSSION THREAD

**Discussion by mbrojo** [valinet/ExplorerPatcher#1647] [Q&A]
**Symptoms:**
I have version 22000.469.41.16 on PC1.
I have version 22261.1555.55.1 on PC2.
On PC1 I have tasks shown only as icons - not grouped.
On PC2 I have tasks shown both as icons and labels - not grouped.

**The problem:**
On PC2 I also want to have only icons. I can't find the settings. What am i doing wrong?

**Comment by vippium** [comment]
share a ss bro....if it is only showing icons, then it is grouped ..else ungrouped

**Comment by pyrates999** [comment]
22000 is windows 11 21h2: https://en.wikipedia.org/wiki/Windows_11_version_history

You should update PC1 to Windows 11 22h2.

And show a screen shot of what you mean.

**Reply by mbrojo** [reply]
PC2 doesnt behave like desired. PC1 is ok.

**Reply by pyrates999** [reply]
PC1 will lose support from Microsoft by October 10, 2023: https://en.wikipedia.org/wiki/Windows_11_version_history

**Comment by mbrojo** [comment]
PC1:
![image](https://github.com/valinet/ExplorerPatcher/assets/31203737/c0532e96-c7c7-45d7-b71e-cb5e32a84f30)

PC2:
![image](https://github.com/valinet/ExplorerPatcher/assets/31203737/949a235f-c114-4cbf-be2d-94f9faad682f)

Here You can see it.

**Comment by pyrates999** [comment]
Here's how to do it: https://www.xda-developers.com/how-enable-taskbar-ungrouping-labels-windows-11/

**Reply by mbrojo** [reply]
It's not working. I joined the dev channel and i cannot see Not combine option. I even checked it on some forums and people say if you dont combine, you see labels...
Thats the problem i am facing right now. I want both - not combine and not show labels.

**Reply by pyrates999** [reply]
How are you going to tell them apart if there's no labels?
